SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -31,6 +31,8 @@ const int FLAME_SENSOR_PIN = 6;
 const int MOTOR_PIN = 7;
 const int COLLISION_SERVO_PIN = 8;
 const int LED_PIN = 12;
+const int  VOLT_PIN = A5;
+float voltage_Now=1.5;
```

Comment:
Left the comment here since i cant make one on line 15 

No new version and contribution